git.md

```diff
@@ -87,6 +106,42 @@ The **footer** should contain any information about breaking changes and is also
 
 Breaking changes should start with the word `BREAKING CHANGE`: with a space or two newlines. The rest of the commit message is then used for this.
 
+
+
+### Pre-commit hook
+
+Commits are usually checked in the project CI, but you can configure [gitlint](https://jorisroovers.com/gitlint/latest/commit_hooks/#pre-commit) as a plugin for [pre-commit](https://pre-commit.com/#installation) hook to validate each commits locally.
+
+Simply install `pre-commit` using `pip`:
```

Comment:
It is recommended to avoid using `simple` or `simply` in a technical documentation. Nothing is obvious for everyone :wink: